.github/scripts/generate_binary_build_matrix.py

```diff
@@ -16,10 +16,10 @@
 CUDA_ARCHES = ["11.8", "12.1", "12.4"]
 
 
-CUDA_ARCHES_FULL_VERSION = {"11.8": "11.8.0", "12.1": "12.1.1", "12.4": "12.4.0"}
+CUDA_ARCHES_FULL_VERSION = {"11.8": "11.8.0", "12.1": "12.1.1", "12.4": "12.4.1"}
```

Comment:
Any reason why those changes are necessary now?

Reply:
docker build fails without this.

```
ERROR: docker.io/nvidia/cuda:12.4.0-cudnn8-devel-ubuntu22.04: not found
```
as there is no image for this ```https://hub.docker.com/layers/nvidia/cuda/12.4.0-cudnn8-devel-ubuntu22.04/images/``` or ```https://hub.docker.com/layers/nvidia/cuda/12.4.0-cudnn-devel-ubuntu22.04/images/```

should we fix this in next PR ?

Reply:
If it's a pre-existing failure let's not try to make any changes here..

Reply:
done